scheduler.go

```diff
@@ -60,8 +60,8 @@ func (s *Scheduler) Less(i, j int) bool {
 	return s.jobs[j].nextRun.Unix() >= s.jobs[i].nextRun.Unix()
 }
 
-// ChangeLoc changes the default time location
-func (s *Scheduler) ChangeLoc(newLocation *time.Location) {
+// SetLocation changes the default time location
+func (s *Scheduler) SetLocation(newLocation *time.Location) {
```

Comment:
This is not backward compatible right?

Reply:
No. However, as this is a brand new repo, we are taking this chance to make some changes to the API before people begin to use it in production

Reply:
:+1: